yarn.lock

```diff
@@ -6461,7 +6466,7 @@ ember-template-imports@^1.1.1:
     ember-cli-version-checker "^5.1.2"
     validate-peer-dependencies "^1.1.0"
 
-ember-template-imports@^3.0.0, ember-template-imports@^3.1.1:
+ember-template-imports@^3.1.1:
```

Comment:
This'll soon be removed from ember-template lint as well:

```
=> Found "ember-template-lint#ember-template-imports@3.4.0"
info This module exists because "_project_#ts-ember-app#ember-template-lint" depends on it.
info Disk size without dependencies: "232KB"
```